src/Symfony/Component/Messenger/Exception/WrappedExceptionsInterface.php

```diff
@@ -19,7 +19,13 @@
 interface WrappedExceptionsInterface extends \Throwable
 {
     /**
+     * @template TClass of class-string<\Throwable>
+     *
+     * @param TClass|null $class
+     *
      * @return \Throwable[]
+     *
+     * @psalm-return (TClass is null ? \Throwable[] : TClass[])
```

Comment:
This is not correct. Since `TClass` is a `class-string`, you're basically saying the this method returns an array of strings, while in fact it's returning an array of objects.

Reply:
You're right, thanks. Fixed in #58068.